tests/integration/kubectl/kubectl_integration_test.go

```diff
@@ -0,0 +1,37 @@
+package kubectl
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+
+	"github.com/gruntwork-io/terratest/modules/k8s"
+	"github.com/gruntwork-io/terratest/modules/random"
+	"github.com/stretchr/testify/require"
+	authv1 "k8s.io/api/authorization/v1"
+)
+
+// An example of verifying the kubectl is operational and that
+// authorization for basic operations.
+func TestKubectlAuthorization(t *testing.T) {
+
```

Comment:
I am not sure that this test is necessary. If kubectl isn't configured properly, I think we can pretty much assume stuff will break.

Reply:
This is checking for authorization w/ `kubectl` for the specific commands being used.  I'm good with removing if you don't see value.  

The idea was to get the permission checks out of the way in some lightweight integration function as opposed to finding out later in one of the more complicated verifications.

Reply:
I am fine with making the assumption that kubectl is properly configured as a prerequisite step.